### PR TITLE
Cosmos parse std tx

### PR DESCRIPTION
**Pull request by kaladinlight**

### CODE REVIEW

File: node/packages/parser/src/cosmos/types.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
+export type Network = 'mainnet'
+
+interface Value {
+  amount: string
+  denom: string
```
Comment (0xApotheosis) [inline node/packages/parser/src/cosmos/types.ts:5]: `denomination`?

File: node/packages/parser/src/cosmos/index.ts [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+      caip2: caip2.toCAIP2({ chain: ChainTypes.Cosmos, network: toNetworkType(this.network) }),
+      confirmations: -1, // TODO: confirmations not tracked by cosmos coinstack
+      status: Status.Confirmed, // no mempool provided by cosmos coinstack currently, and can be inferred from confirmations when added
+      transfers: [],
+      txid: tx.txid,
+      value: tx.value,
+    }
+
+    // messages make best attempt to track where value is transferring to for a variety of tx types
+    // logs provide more specific information if needed as mor complex tx types are added
```
Comment (0xApotheosis) [inline node/packages/parser/src/cosmos/index.ts:43]:
```suggestion
    // logs provide more specific information if needed as more complex tx types are added
```

File: node/packages/parser/src/cosmos/index.ts [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    tx.messages.forEach((msg) => {
```
Comment (0xApotheosis) [inline node/packages/parser/src/cosmos/index.ts:44]: I'd like to see us replace this mutating pattern, which we use in many places, with a pure one (e.g. `return tx.messages.map...`, with the usage of `aggregateTransfer` refactored to be pure, too) - fine for now though.

**Review by 0xApotheosis — APPROVED** [review]

File: node/packages/parser/src/cosmos/types.ts (hunk 1 above)
Comment (kaladinlight) [reply node/packages/parser/src/cosmos/types.ts:5]: Original key name from cosmos-sdk that I am passing back through unchained, going to leave it as.

File: node/packages/parser/src/cosmos/index.ts (hunk 3 above)
Comment (kaladinlight) [reply node/packages/parser/src/cosmos/index.ts:44]: Ticket created: https://github.com/shapeshift/unchained/issues/268

**Pull request merged**